rwslib/builders/__init__.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'glow'
+
+# Common elements
+from rwslib.builders.common import ODMElement, TransactionalElement
+
+# Constants
+from rwslib.builders.constants import *
```

Comment:
yeah, this is the height of expediency; I need to be able to keep the `from builders import *` pattern, while separating out into sub-modules.  Will chat next time in the office.